test/end2end_test.go

```diff
@@ -53,6 +53,7 @@ import (
 	_ "google.golang.org/grpc/encoding/gzip"
 	_ "google.golang.org/grpc/grpclog/glogger"
 	"google.golang.org/grpc/health"
+	healthgrpc "google.golang.org/grpc/health/grpc_health_v1"
```

Comment:
Nit: proto imports should end in "pb".

Reply:
in keeping with alts pattern (PR linked in desc) I am using pb to indicate messages, and grpc to indicate grpc stubs

Reply:
Which PR are you referring to?